.github/workflows/storybook-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Install Playwright 🎭
         run: npx playwright install --with-deps
       - name: Run Storybook Tests 📚
-        run: npm run build:storybook --quiet
```

Comment:
Why do we remove  this flag?

Reply:
Not really trying to remove it. The tests are not running properly and I am trying to debug it 😩